textdb/textdb-dataflow/src/main/java/com/google/re2j/PublicParser.java

```diff
@@ -0,0 +1,21 @@
+package com.google.re2j;
+
+/**
+ * Wrapper class of re2j.Parser.<!-- --> This class parses a regex string to a PublicRegexp object. <br>
```

Comment:
After Javadoc is generated, it shows the first sentence of each class in the overview page. And Javadoc treats dot as the end of each sentence. I want the whole sentence "Wrapper class of re2j.Parser.This class parses a regex string to a PublicRegexp object." to appear in the overview. So I have to add  "<!-- -->"(a html comment) to tell Javadoc don't treat this dot as the end of this sentence. Adding something else like "&nbsp"(space in html meta character) will also work.